<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Quasi</class>
 <widget class="QWidget" name="Quasi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="lblResolution">
       <property name="text">
        <string>Resolution:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="MantidQt::MantidWidgets::DataSelector" name="dsSample">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="loadLabelText" stdset="0">
        <string>Plot</string>
       </property>
       <property name="workspaceSuffixes" stdset="0">
        <stringlist>
         <string>_red</string>
         <string>_sqw</string>
        </stringlist>
       </property>
       <property name="fileBrowserSuffixes" stdset="0">
        <stringlist>
         <string>_red.nxs</string>
         <string>_sqw.nxs</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblInput">
       <property name="text">
        <string>Input:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="MantidQt::MantidWidgets::DataSelector" name="dsResolution">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoLoad">
        <bool>true</bool>
       </property>
       <property name="workspaceSuffixes" stdset="0">
        <stringlist>
         <string>_res</string>
         <string>_red</string>
         <string>_sqw</string>
        </stringlist>
       </property>
       <property name="fileBrowserSuffixes" stdset="0">
        <stringlist>
         <string>_res.nxs</string>
         <string>_red.nxs</string>
         <string>_sqw.nxs</string>
        </stringlist>
       </property>
       <property name="showLoad">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbProgram">
       <item>
        <property name="text">
         <string>Lorentzians</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Stretched Exponential</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblProgram">
       <property name="text">
        <string>Program</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="fitOptions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Fit Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QCheckBox" name="chkFixWidth">
          <property name="text">
           <string>Fix Width</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="chkUseResNorm">
          <property name="text">
           <string>Use ResNorm</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="MantidQt::MantidWidgets::MWRunFiles" name="mwFixWidthDat">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="label" stdset="0">
           <string/>
          </property>
          <property name="fileExtensions">
           <stringlist>
            <string>.dat</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="cbBackground">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Sloping</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Flat</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Zero</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="chkElasticPeak">
            <property name="text">
             <string>Elastic Peak</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkSequentialFit">
            <property name="text">
             <string>Sequential Fit</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <widget class="MantidQt::MantidWidgets::DataSelector" name="dsResNorm">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="autoLoad">
           <bool>false</bool>
          </property>
          <property name="workspaceSuffixes" stdset="0">
           <stringlist>
            <string>_ResNorm</string>
           </stringlist>
          </property>
          <property name="fileBrowserSuffixes" stdset="0">
           <stringlist>
            <string>_ResNorm.nxs</string>
           </stringlist>
          </property>
          <property name="showLoad">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lblBackground">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Background:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="treeSpace"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="plotSpace"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_44">
     <item>
      <widget class="QGroupBox" name="gbOutput">
       <property name="title">
        <string>Output Options</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblPlotResult">
          <property name="text">
           <string>Plot Result: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbPlot">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Amplitude</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FWHM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fit</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Prob</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_18">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chkSave">
          <property name="text">
           <string>Save Result</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::MWRunFiles</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/MWRunFiles.h</header>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/DataSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
